src/skmatter/_selection.py

```diff
@@ -92,13 +93,28 @@ def __init__(
         score_threshold_type="absolute",
         progress_bar=False,
         full=False,
-        random_state=0,
+        random_state=1,
```

Comment:
Usually it is None, I don't think it is important if we use 0, but the change to 1 I dont get.

Reply:
Fine to change it back.
```suggestion
        random_state=0,
```